R/deltaccd.R

```diff
@@ -5,7 +5,8 @@ NULL
 
 
 globalVariables(c(
-  'i', 'groupNow', 'group', '.', 'gene1', 'gene2', 'rho', 'group2Now', '.SD'))
+  'i', 'groupNow', 'group', '.', 'gene1', 'gene2', 'rho', 'group2Now',
+  'ematNow', '.SD', 'variance'))
```

Comment:
Are you sure ematNow needs to be treated like a global variable?

Reply:
`devtools::check` says no, but then I guess I don't really understand when _do_ you need to declare a global variable?

Reply:
Ideally never, but in practice we do it for columns that get treated like variables by data.table or iterated variables in a foreach loop, since R's package check doesn't realize those aren't actually global variables.